6.1/6.1/Program.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace _6._1
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите текст на английском:");
+            string TextEnglish = Console.ReadLine();
+            Console.WriteLine("Вот перевод вашего текста на Leet: " + LeetToTranslate(TextEnglish));
+        }
+
+        static string LeetToTranslate(string SomeTxt)
+        {
+            SomeTxt = SomeTxt.ToUpper();
+            return SomeTxt.Replace("A", "4")
```

Comment:
Для параметров методов принято использовать имена, начинающиеся с маленькой буквы. Исходный текст лучше не изменять. Также все можно сделать в одну цепочку.
```suggestion
        static string LeetToTranslate(string someTxt)
        {
            return someTxt.ToUpper()
           .Replace("A", "4")
```